### DISCUSSION TITLE
Run .net 8 dotnet monitor as a sidecar in AWS ECS

### DISCUSSION THREAD

**Discussion by sbrink-zoll** [dotnet/dotnet-monitor#7054] [Q&A]
I'm trying to follow the kubernetes.md example for setting up dotnet monitor to run as a sidecar and apply it to a AWS ECS service instead of kubernetes.  No matter what I try I can't seem to get past a permission denied error.  It seems to be throwing this when trying to open the socket using the shared volume:
{"Timestamp":"2024-07-29T18:40:55.1869517Z","EventId":11,"LogLevel":"Error","Category":"Microsoft.Extensions.Hosting.Internal.Host","Message":"Hosting failed to start","Exception":"System.Net.Sockets.SocketException (13): Permission denied\n at System.Net.Sockets.Socket.DoBind(EndPoint endPointSnapshot, SocketAddress socketAddress)\n at System.Net.Sockets.Socket.Bind(EndPoint localEP)\n at Microsoft.Diagnostics.NETCore.Client.IpcUnixDomainSocket.Bind(IpcUnixDomainSocketEndPoint localEP)\n at Microsoft.Diagnostics.NETCore.Client.IpcUnixDomainSocketServerTransport.CreateNewSocketServer()\n at Microsoft.Diagnostics.NETCore.Client.IpcUnixDomainSocketServerTransport..ctor(String path, Int32 backlog, IIpcServerTransportCallbackInternal transportCallback)\n at Microsoft.Diagnostics.NETCore.Client.IpcServerTransport.Create(String address, Int32 maxConnections, Kind kind, IIpcServerTransportCallbackInternal transportCallback)\n at Microsoft.Diagnostics.NETCore.Client.ReversedDiagnosticsServer.Start(Int32 maxConnections)\n at Microsoft.Diagnostics.Tools.Monitor.ServerEndpointInfoSource.ExecuteAsync(CancellationToken stoppingToken)\n at Microsoft.Diagnostics.Tools.Monitor.ServerEndpointInfoSource.ExecuteAsync(CancellationToken stoppingToken)\n at Microsoft.Extensions.Hosting.Internal.Host.\u003CStartAsync\u003Eb__15_1(IHostedService service, CancellationToken token)\n at Microsoft.Extensions.Hosting.Internal.Host.ForeachService[T](IEnumerable\u00601 services, CancellationToken token, Boolean concurrent, Boolean abortOnFirstException, List\u00601 exceptions, Func\u00603 operation)","State":{"Message":"Hosting failed to start","{OriginalFormat}":"Hosting failed to start"},"Scopes":[]}

Attached is the json from AWS to set this up.  Has anyone been able get this to work?

[dotnet-monitor-test-revision2.json](https://github.com/user-attachments/files/16418121/dotnet-monitor-test-revision2.json)

**Comment by birojnayak** [comment]
@sbrink75 are you still facing the issue AWS/ECS ?
